README.md

```diff
@@ -117,8 +117,16 @@ alias FUCK='fuck'
 Or in `config.fish`:
 
 ```fish
-function fuck
-    eval (thefuck $history[1])
+function __thefuck_repl --description 'Replace operators into fish-compatible'
+    set -l tmp (echo $argv | sed 's/ && / ; and /g')
+    echo $tmp | sed 's/ || / ; or /g'
+end
+
+function fuck --description 'Correct your previous console command'
+    set -l eval_script /tmp/.thefuck.fish
```

Comment:
Absolutely, @daenney! I'll update this pull request ASAP. Thank you!
Em 26/04/2015 06:20, "Daniele Sluijters" <notifications@github.com>
escreveu:

> In README.md
> <https://github.com/nvbn/thefuck/pull/130#discussion_r29108049>:
>
> > @@ -117,8 +117,16 @@ alias FUCK='fuck'
> >  Or in `config.fish`:
> >
> >  ```fish
> > -function fuck
> > -    eval (thefuck $history[1])
> > +function __thefuck_repl --description 'Replace operators into fish-compatible'
> > +    set -l tmp (echo $argv | sed 's/ && / ; and /g')
> > +    echo $tmp | sed 's/ || / ; or /g'
> > +end
> > +
> > +function fuck --description 'Correct your previous console command'
> > +    set -l eval_script /tmp/.thefuck.fish
>
> This should probably use $TMPDIR instead / when it is set.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nvbn/thefuck/pull/130/files#r29108049>.
>